workshops/static/online_country.js

```diff
@@ -0,0 +1,42 @@
+var prev_venue,
+    prev_address,
+    prev_latitude,
+    prev_longitude,
+    online_selected_before = false;
+
+var id_country = '#id_country, #id_event-country',
+    id_venue = '#id_venue, #id_event-venue',
+    id_address = '#id_address, #id_event-address',
+    id_latitude = '#id_latitude, #id_event-latitude',
+    id_longitude = '#id_longitude, #id_event-longitude';
+
+// disable latitude, longitude inputs if online country selected
+$(id_country).change(function(e) {
+  var new_country = $(e.target).val();
+
+  if (new_country == 'W3') {
+    prev_venue = $(id_venue).val();
+    prev_address = $(id_address).val();
+    prev_latitude = $(id_latitude).val();
+    prev_longitude = $(id_longitude).val();
+    $(id_venue).val('Internet');
+    $(id_address).val('Internet');
+    $(id_latitude).val('-48.876667');
+    $(id_longitude).val('-123.393333');
+    $(id_venue).prop('disabled', true);
+    $(id_address).prop('disabled', true);
+    $(id_latitude).prop('disabled', true);
+    $(id_longitude).prop('disabled', true);
```

Comment:
Ah, this keeps admins from tweaking after the JavaScript fills in the values, so I think keep the JavaScript here, but also add the `Event.save()` logic (e.g. folks could use cURL to POST, or be saving events from a Django shell, or …).